sim/shaman/fire_elemental_spells.go

```diff
@@ -61,18 +61,19 @@ func (fireElemental *FireElemental) registerFireNova() {
 		DamageMultiplier: 1,
 		CritMultiplier:   2, //Estimated from beta testing
 		ThreatMultiplier: 1,
+		BonusCoefficient: 1.00,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
 			for _, aoeTarget := range sim.Encounter.TargetUnits {
-				baseDamage := sim.Roll(1527, 1731) * sim.Encounter.AOECapMultiplier() //Estimated from beta testing
+				baseDamage := sim.Roll(1370, 1574) * sim.Encounter.AOECapMultiplier() //Estimated from beta testing
```

Comment:
Looks like you used 157 SP to get to the new value, i'm not sure if fire ele also have the first 10 int not giving spell power or not

Reply:
now that i think about it, it should adds up to the same damage when inheriting stats from the owner since you get back the 10 SP that you previously lost because of no base int

Reply:
yes